docs/frontity-cli/serve.md

```diff
@@ -0,0 +1,25 @@
+# `serve`
+
+Starts a server in production mode.
```

Comment:
Let's say that it starts a frontity server in production mode. Perhaps obvious to us but maybe not so for a beginner.